go.mod

```diff
@@ -5,11 +5,11 @@ go 1.23.1
 require (
 	github.com/corazawaf/coraza-coreruleset v0.0.0-20240226094324-415b1017abdc
 	github.com/corazawaf/coraza/v3 v3.2.1
+	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/dropmorepackets/haproxy-go v0.0.5
 	github.com/jcchavezs/mergefs v0.0.0-20230503083351-07f27d256761
 	github.com/magefile/mage v1.15.0
 	github.com/mccutchen/go-httpbin/v2 v2.14.0
-	github.com/pires/go-proxyproto v0.8.0
```

Comment:
Why does this get yeetet :thinking:

Reply:
It tells me that it is unused 🤔 

```shell
go mod why github.com/pires/go-proxyproto
# github.com/pires/go-proxyproto
(main module does not need package github.com/pires/go-proxyproto)
```

Anyways, I put it back, it isn't the scope of this PR